guides/v2.1/config-guide/bootstrap/magento-modes.md

```diff
@@ -93,6 +100,15 @@ In production mode:
 -   Static view files are not materialized and URLs for them are composed on the fly. Static view files are served from the {% glossarytooltip 0bc9c8bc-de1a-4a06-9c99-a89a29c30645 %}cache{% endglossarytooltip %} only.)
 -   Errors are logged to the file system and are never displayed to the user.
 
+## Maintenance mode
+
+Run the Magento software in maintenance mode when you need to take your site offline, for example when you are performing installation and upgrade tasks. When your site is in maintenance mode, normal site visitors get a `Service Temporarily Unavailable` message when they navigate to your store. You can configure maintenance mode to allow visitors from authorized IP addresses to view the store normally.
+
+You can [enable and disable maintenance mode]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-maint.html) and configure exempt IP addresses using Magento CLI commands.
+
+For customers using {{site.data.var.ece}}, maintenance mode is enabled automatically for the duration of the deploy phase. When the deployment completes sucessfully, maintenance mode is turned off, and the Magento application is put back into production mode. See [Deployment hooks]({{ page.baseurl }}/cloud/reference/discover-deploy.html).
```

Comment:
Passive. Previously, the word "run" is used to describe the mode activation. So, in keeping with this spirit, maybe say
"...Magento runs in maintenance mode during the deploy phase."

Also, "turned on" and "put back" sounds like you're explaining an old tube television. Maybe.. "When the deployment completes successfully, Magento returns to running in normal production mode."

Also, Magento application sounds better, but wonder if we should use a variable.

Also, successfully is misspelled.